packages/tables/resources/views/components/selection-indicator.blade.php

```diff
@@ -18,13 +18,13 @@ class="inline-block w-4 h-4 mr-3 rtl:mr-0 rtl:ml-3 text-primary-500"
 
     <span id="{{ $this->id }}.table.selection.indicator.record-count.{{ $allRecordsCount }}" x-show="{{ $allRecordsCount }} !== selectedRecords.length">
         <button x-on:click="selectAllRecords" class="text-sm font-medium text-primary-600">
-            {{ __('tables::table.selection_indicator.buttons.select_all.label', ['count' => $allRecordsCount]) }}.
+            {{ trans_choice('tables::table.selection_indicator.buttons.select_all.label', $allRecordsCount, ['count' => $allRecordsCount]) }}.
         </button>
     </span>
 
     <span>
         <button x-on:click="deselectAllRecords" class="text-sm font-medium text-primary-600">
-            {{ __('tables::table.selection_indicator.buttons.deselect_all.label') }}.
+            {{ trans_choice('tables::table.selection_indicator.buttons.deselect_all.label', $allRecordsCount, ['count' => $allRecordsCount]) }}.
```

Comment:
`$allRecordsCount` isn't the currently selected number of records, that is `selectedRecords.length` in JavaScript

Reply:
Reverted for deselect